item_mapper.rb

```diff
@@ -0,0 +1,21 @@
+class ItemMapper
+  class << self
+    def from_structs(items)
+      items.map { |item| klass_for(item).new(item.name, item.sell_in, item.quality) }
+    end
+
+    private
+    def klass_for(item)
+      Module.const_get(klass_hash[item.name] || 'StandardItem')
+    end
+
+    def klass_hash
+      {
+        'Aged Brie' => 'Cheese',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.